Common.cmake

```diff
@@ -128,6 +129,12 @@ if(NOT TARGET ${PROJECT_NAME}-all)
   set_target_properties(${PROJECT_NAME}-all PROPERTIES FOLDER ${PROJECT_NAME})
 endif()
 
+add_custom_target(${PROJECT_NAME}-install
+  ${CMAKE_COMMAND} -P ${PROJECT_BINARY_DIR}/cmake_install.cmake
+  DEPENDS ${PROJECT_NAME}-all)
+set_target_properties(${PROJECT_NAME}-install PROPERTIES
+  EXCLUDE_FROM_DEFAULT_BUILD ON FOLDER ${PROJECT_NAME}/doxygen)
```

Comment:
Oh, I got confused by this in the past too ;-)
It should not be read as "ON FOLDER" but as: "EXCLUDE_FROM_DEFAULT_BUILD TRUE" "FOLDER ${PROJECT_NAME}/tests".
+1, FOLDER ${PROJECT_NAME}/doxygen does not look right (C&P error)